code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -60,7 +60,13 @@ func (sc *StorageSmartContract) addAllocation(alloc *StorageAllocation,
 			"saving new allocation in db: %v", err)
 	}
 
-	buff := alloc.Encode()
+	type Output struct {
```

Comment:
should we instead define a method something like `GetOutput()` on StorageAllocation and use that instead? In future, if we need to add a similar logic for other APIs, we can do so?